Mail/Views/Thread/MessageBodyView.swift

```diff
@@ -83,19 +84,23 @@ struct MessageBodyView: View {
     }
 
     private func loadBody(blockRemoteContent: Bool) {
-        Task {
+        Task.detached {
```

Comment:
`Task.detached` (and `MainActor.run`) are not needed since `loadHTMLString` already runs on a background thread

Reply:
yeah it is overkill here I agree.

Reply:
done